src/main/java/jesseg/ibmi/opensource/nginx/NginxConf.java

```diff
@@ -46,7 +48,7 @@ public static void main(final String[] args) {
 
         NginxConf root;
         try {
-            root = new NginxConf(new File("C:\\nginx.in"));
+            root = new NginxConf(new File("C:\\nginx.in"), null);
```

Comment:
Rather than `null`, we can use a defaulty one like here: https://github.com/ThePrez/ServiceCommander-IBMi/blob/51cce1dbab3f680eb495cf284e8729447c2fbcef/src/main/java/jesseg/ibmi/opensource/ServiceCommander.java#L204

Reply:
I added a default logger, with verbose set to false